src/components/pages/LockScreen.tsx

```diff
@@ -53,10 +53,46 @@ export const LockScreen = (): JSX.Element => {
   const biometryAvailable = Boolean(flagshipMetadata.biometry_available)
   const biometryType = flagshipMetadata.biometry_type
 
-  const onBiometryLock = (): void =>
-    void handleChange(setBiometry, 'biometryLock', webviewIntent).then(
-      value => value && setAutoLock(true)
-    )
+  const onBiometryLock = (): void => {
+    if (!webviewIntent) return
```

Comment:
nit: Do we really need this check here? Shouldn't be done before within the Settings' app?

Reply:
Due to historical instability with cozy-intent I usually go defensive programming whenever using it